pkg/internal/ebpf/common/kafka_detect_transform_test.go

```diff
@@ -23,7 +23,27 @@ func TestProcessKafkaRequest(t *testing.T) {
 			},
 		},
 		{
-			name:  "Produce request",
+			name:  "Fetch request (v12)",
```

Comment:
Can we make a test where the string doesn't have `\t` inside, so the split doesn't work?

Reply:
Also perhaps a test where there's no `\x02\x00`?

Reply:
Yeah, just manually crafted would be great. The reason I mentioned this is because there are so many traps here with the kafka protocol, I'd like to make the code as defensive as possible.

Reply:
I ended testing the function to extract the topic itself, do you think is enough to test this?